src/styles/blocks/_nav.scss

```diff
@@ -0,0 +1,51 @@
+@use "../units/reset";
```

Comment:
import all the files to main.scss. if you have problems with @use, use @import instead